pkg/orchestrator/connectors.go

```diff
@@ -163,3 +165,30 @@ func (c *ConnectorOrchestrator) Update(ctx context.Context, id string, config co
 	r.Skip()
 	return conn, nil
 }
+
+func (c *ConnectorOrchestrator) Validate(
+	ctx context.Context,
+	t connector.Type,
+	config connector.Config,
+) error {
+	logger := log.CtxLogger{Logger: zerolog.Nop()}
+	d, err := c.plugins.NewDispenser(logger, config.Plugin)
```

Comment:
is it better to get the dispenser here, or in the plugin service?

Reply:
I'd say plugin service.

Reply:
I just realized you already did it this way, let's leave it like this.

Reply:
I did this way to minimize duplicated code